ansible/sepolia.yml

```diff
@@ -17,10 +17,10 @@
     - nimbus-sepolia-linux
   roles:
     - { role: update-netplan,           tags: [ update-netplan ], when: ansible_system == 'Linux' }
-    - { role: infra-role-open-ports,    tags: [ open-ports ]        }
-    - { role: infra-role-smart-metrics, tags: [ smart-metrics ]     }
-    - { role: redirect-ports,           tags: [ redirect-ports ]    }
-    - { role: nimbus-era-files,         tags: [ nimbus-era-files ]  }
+    - { role: infra-role-open-ports,    tags: [ open-ports ]                                      }
+    - { role: infra-role-smart-metrics, tags: [ smart-metrics ]                                   }
+    - { role: redirect-ports,           tags: [ redirect-ports ]                                  }
+    - { role: nimbus-era-files,          tags: [ nimbus-era-files ]                                 }
```

Comment:
Wrong indent.

Reply:
this is very weird, looks okay in my editor

<img width="856" alt="Screenshot 2024-07-30 at 5 29 02 PM" src="https://github.com/user-attachments/assets/8a63f009-8c47-4994-bd06-e8b1c080939f">

However, looks messed up in github UI

<img width="893" alt="Screenshot 2024-07-30 at 5 29 22 PM" src="https://github.com/user-attachments/assets/7cd7de49-5523-4c57-b084-3d303a6743ff">

Reply:
looks messed up in vim as well, so I shall fix
<img width="1368" alt="Screenshot 2024-07-30 at 5 30 46 PM" src="https://github.com/user-attachments/assets/1d41a8d9-7800-47f7-afa2-b2b62af5c5b0">